src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -320,7 +321,8 @@ public function getAttributes()
     {
         return [
             'data-editor' => 'tinyMCE', // Register ss.editorWrappers.tinyMCE
-            'data-config' => Convert::array2json($this->getConfig())
+            'data-config' => Convert::array2json($this->getConfig()),
+            'style' => 'height:350px'
```

Comment:
don't need the `style` attribute if you're already setting `min_height` in the config setting already :D